core/themes/basis/theme-settings.php

```diff
@@ -70,3 +70,95 @@
     $form['footer'][$field] = color_get_color_element($form['theme']['#value'], $field, $form);
   }
 }
+
+/**
+ * Altering the theme settings for CSS updates.
+ */
+function basis_form_system_theme_settings_alter(&$form, &$form_state, $form_id = NULL) {
+  // Ensure template.php is included.
+  $theme_name = $form['theme']['#value'];
+  $theme_path = backdrop_get_path('theme', 'basis');
+  require_once $theme_path . '/template.php';
+  $install_version = config_get('system.core', 'install_version');
+
+  $css_update_options = array(
+    'no_updates' => t('Default'),
+    'all_updates' => t('All updates'),
+    'custom' => t('Specific version'),
+  );
+
+  $form['css_updates'] = array(
+    '#type' => 'details',
+    '#open' => FALSE,
+    '#summary' => t('CSS Updates'),
+    '#details' => t('There are occasionally updates available for Basis that may affect existing sites, changing this setting may cause unexpected changes to your theme. For more information and details, see the <a href="https://docs.backdropcms.org/documentation/layouts-and-templates" target="_blank">online documentation</a>.</br></br>Select which updates you would like to accept. '),
+    '#attributes' => array(
+      'class' => array('description'),
+    ),
+  );
+
+  // Load the saved configuration value.
+  $form['css_updates']['css_update'] = array(
+    '#title' => t('CSS Update Options'),
+    '#type' => 'radios',
+    '#options' => $css_update_options,
+    '#default_value' => theme_get_setting('css_update', $theme_name),
+    '#config' => 'basis.settings',
+    'no_updates' => array(
+      '#description' => t('Accept no updates introduced after installation.'),
+    ),
+    'all_updates' => array(
+      '#description' => t('Warning: Some future changes may effect your site.'),
+    ),
+    'custom' => array(
+    ),
+  );
+
+  $default_version_value = theme_get_setting('css_update_version', $theme_name);
+
+  // Reverse lookup to get the correct key.
+  $options = basis_supplemental_css_versions();
+  $default_version_key = array_search($default_version_value, $options, TRUE);
+
+  // Conditionally displayed CSS updates select element.
+  $form['css_updates']['css_update_version'] = [
```

Comment:
short array syntax is used here `= [` use here. Can we switch back to `array(`

Reply:
Fixed